browser/notifications/notification_platform_bridge_brave_custom_notification.cc

```diff
@@ -0,0 +1,178 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/notifications/notification_platform_bridge_brave_custom_notification.h"
+
+#include <memory>
+#include <string>
+
+#include "base/bind.h"
+#include "base/callback.h"
+#include "base/memory/scoped_refptr.h"
+#include "base/run_loop.h"
+#include "base/task/post_task.h"
+#include "brave/components/brave_ads/browser/ads_notification_handler.h"
+#include "brave/components/brave_ads/browser/ads_service.h"
+#include "brave/components/brave_ads/browser/ads_service_factory.h"
+#include "brave/components/brave_ads/browser/ads_service_impl.h"
+#include "brave/ui/brave_custom_notification/message_popup_view.h"
+#include "brave/ui/brave_custom_notification/public/cpp/notification.h"
+#include "chrome/browser/browser_process.h"
```

Comment:
this header list does not appear to be correct. I don't see any references from browser_process.h and you're missing browser_context.h just as a few examples. Please ensure that you include what you use and only what you use https://github.com/include-what-you-use/include-what-you-use/blob/master/docs/WhyIWYU.md